node/src/components/storage.rs

```diff
@@ -358,23 +191,18 @@ pub struct Storage {
     /// The state storage database.
     #[data_size(skip)]
     state_store_db: Database,
-    /// A map of block height to block ID.
-    block_height_index: BTreeMap<u64, BlockHash>,
-    /// The height of the highest block from which this node has an unbroken sequence of full
-    /// blocks stored (and the corresponding global state).
-    lowest_available_block_height: u64,
     /// Highest block available when storage is constructed.
     highest_block_at_startup: u64,
-    /// A map of era ID to switch block ID.
-    switch_block_era_id_index: BTreeMap<EraId, BlockHash>,
-    /// A map of deploy hashes to hashes of blocks containing them.
-    deploy_hash_index: BTreeMap<DeployHash, BlockHash>,
+    /// Various indices used by the component.
+    #[data_size(skip)]
+    indices: RwLock<Indices>,
     /// Whether or not memory deduplication is enabled.
     enable_mem_deduplication: bool,
     /// An in-memory pool of already loaded serialized items.
     ///
     /// Keyed by serialized item ID, contains the serialized item.
-    serialized_item_pool: ObjectPool<Box<[u8]>>,
+    #[data_size(skip)]
```

Comment:
Why are we skipping this now? Isn't it true that it may contribute to majority of the memory this component uses?

Reply:
Good catch.

Reply:
I thought so as well. Can you add comments then?